src/main/java/pl/pszczolkowski/mustdo/domain/task/bo/TasksListBO.java

```diff
@@ -0,0 +1,15 @@
+package pl.pszczolkowski.mustdo.domain.task.bo;
+
+import pl.pszczolkowski.mustdo.domain.task.dto.TasksListSnapshot;
+
+public interface TasksListBO {
+
+   TasksListSnapshot add(String name, Long boardId);
+
+   TasksListSnapshot rename(Long id, String name);
```

Comment:
Yeah and when I told you that in GrowUp project you said that entity isn't returning anything so it's ok....